src/config/configStore.ts

```diff
@@ -115,7 +115,7 @@ export abstract class BaseConfigStore<
    * If the value is an object, a clone will be returned.
    */
   public get<K extends Key<P>>(key: K, decrypt?: boolean): P[K];
-  public get<T = ConfigValue>(key: string, decrypt?: boolean): T;
```

Comment:
renaming the type letter because there's already a `T` at the top level, and this doesn't refer to that (no-shadow)